src/components/Agreements/AgreementForm/Sections/AgreementFormExternalLicenses.js

```diff
@@ -1,12 +1,26 @@
 import React from 'react';
 import { FormattedMessage } from 'react-intl';
 import { FieldArray } from 'redux-form';
+import PropTypes from 'prop-types';
 
 import { Accordion } from '@folio/stripes/components';
 import { DocumentsFieldArray } from '@folio/stripes-erm-components';
 
 export default class AgreementFormExternalLicenses extends React.Component {
+  static propTypes = {
+    handlers: PropTypes.shape({
+      onDeleteFile: PropTypes.func.isRequired,
+      onDownloadFile: PropTypes.func.isRequired,
+      onUploadFile: PropTypes.func.isRequired,
+    }),
+  };
+
+  getDeleteFilePath = (file) => `/agreements/files/${file.id}`;
```

Comment:
LOL you don't need these; it does show me that I left these two functions in my Licenses code as a refactor artifact.

Reply:
Cleaned it up